release/build_release.sh

```diff
@@ -146,6 +160,8 @@ function main {
 		lc_time_run report_patcher_status
 	fi
 
+	lc_time_run create_release_properties_file
```

Comment:
Please wait for this to get merged, before resending the PR so we can use the final version. I know it's blocked on me currently.